crates/task-impls/src/request.rs

```diff
@@ -163,7 +168,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, Ver: StaticVersionType + 'st
             delay: self.delay,
             recipients,
         };
-        let Ok(data) = Serializer::<Ver>::serialize(&request) else {
+        let Ok(data) = bincode_opts().serialize(&request) else {
```

Comment:
Hmm this should work with versioned Serializer.  Maybe the deserailize side needs to change instead?